### ISSUE TITLE
Grafana update, brak danych

### ISSUE THREAD

**Issue by kzajac83** [tcich/ha-addon-energa-meter#16]
Cześć. 

Mam dane do wykresu pod moim lokalnym adresem:

http://192.168.15.20:8000/charts

Natomiast po dzisiejszym update'ie dodatku Grafana nie ma żadnych danych, nawet historycznych. Ogólnie nie znam się na Grafanie i wszystko oparłem na instrukcji z tego repozytorium.

Ktoś podpowie co i jak sprawdzić? Teraz wygląda to tak:

![Screenshot_2024-11-12-12-02-15-629_io homeassistant companion android](https://github.com/user-attachments/assets/76c36d4a-c4dd-4087-a640-83dea31e2b06)

Pozdrawiam, Krzysiek

**Comment by paki111** [comment]
Niestety u mnie dziś też to samo.
Podejrzewam jednak, że to nie wina aktualizacji grafany. Jak wcześniej miałem dane do wykresu pod adresem http://[192.168.0.17:8000/charts?start_date...](http://192.168.0.17:8000/charts?start_date=1695332400129&end_date=1697924583285&mp=244850&zone=1) to teraz mam pusty obiekt `{
    "charts": []
}`

**Comment by kzajac83** [comment]
Ja akurat dane mam, tylko grafana zrobiła się pusta


![Screenshot_2024-11-12-14-16-34-213_com brave browser-edit](https://github.com/user-attachments/assets/a15a048b-97e5-4353-b5ad-ff819957f516)

**Comment by paki111** [comment]
Na drugim koncie, gdzie mam tylko 1 licznik i mało danych historycznych, to działa. Tylko na koncie gdzie mam dużo liczników i dużo danych historycznych jest ten problem z pustym JSONEM http://192.168.0.17:8000/charts?mp=...
Może to kwestia strumienia danych z serwera [mojlicznik](https://mojlicznik.energa-operator.pl/)
Mam przeczucie, że to tymczasowy problem i wszystko wróci do normy.

**Comment by paki111** [comment]
> Ja akurat dane mam, tylko grafana zrobiła się pusta
> 
> Screenshot...

Zapomniałem dodać, że musiałem w grafanie zainstalować ponownie plugin JSON API. Może nałożyły się 2 problemy - brak pluginu w grafanie po aktualizacji i tymczasowy brak danych z https://mojlicznik.energa-operator.pl/

**Comment by kzajac83** [comment]
> Zapomniałem dodać, że musiałem w grafanie zainstalować ponownie plugin JSON API

Tak, znikał ten dodatek i chyba cała konfiguracja w Grafanie po aktualizacji :(

**Comment by kzajac83** [comment]
Po dograniu dodatku nic się nie zmieniło. Dopiero po restarcie dodatku Energa Meter Grafana pokazała układ współrzędnych, ale bez danych. Jak wyzumowałem dane daleko poza dziś to dopiero coś się pokazało. 

Chyba rzeczywiście dwa problemy się nałożyły

![Screenshot_2024-11-12-15-38-43-870_io homeassistant companion android](https://github.com/user-attachments/assets/5b884e8c-b212-4584-a613-9e13b3c520cd)

Update:
Ok, dane zaczęły spływać, chyba ruszyło. Na razie mam bieżące dane dotyczące zużycia, brak produkcji bieżącej, za pewne spłyną bo widzę już za zeszły rok.

**Comment by paki111** [comment]
Wszystkie problemy ustąpiły. Po dłuższej przerwie dostęp do danych powrócił i dashboardy w Grafanie ponownie działają.

**Comment by kzajac83** [comment]
U mnie też w wszystko już śmiga.  

Wiem, że to nie repozytorium Grafany,ale zastanawia mnie czy to normalne, że dodatek Grafana do HA gubi swoje pluginy po update?

**Comment by paki111** [comment]
> Wiem, że to nie repozytorium Grafany,ale zastanawia mnie czy to normalne, że dodatek Grafana do HA gubi swoje pluginy po update?

Nadal się to dzieje. Z niewiadomych przyczyn, co jakiś czas trzeba instalować ponownie pluginy od źródeł danych w Grafania.

**Comment by paki111** [comment]
Można już chyba zamknąć ten wątek. Problem przestał się pojawiać. Albo zostało to rozwiązane przez którąś aktualizację, albo fakt że  zrezygnowałem z dodatku w HA i postawiłem Grafanę oddzielnie.

**Comment by kzajac83** [comment]
U mnie tak było przez jakiś czas, a teraz powrócił, ale tak, to bym zamknął bo to chyba problem Grafany lub HA, a nie tego dodatku
